### ISSUE TITLE
Submission form

### ISSUE THREAD

**Issue by nschneid** [dimsum16/dimsum16.github.io#1]
An online form for system submission would help us keep track of everything. Some ideas on questions:
- Team name
- Contact name
- Contact email
- System name (specify this if submitting multiple systems)
- What resources were used to run and/or train the system? The answer to this question will determine the data condition the system competes in.
  - [ ] Task training data
  - [ ] English WordNet
  - [ ] The distributed word clusters
  - [ ] The Yelp Academic dataset
  - [ ] Other lexicons (specify: )
  - [ ] Other corpora (specify: )
  - [ ] A syntactic parser (specify: e.g., Stanford Parser 3.5.1 English Basic Dependencies)
  - [ ] A semantic parser or semantic role labeler (specify: e.g., SEMAFOR 2.1 with TurboParser 2.2)
  - [ ] Other: (specify)
- [ ] We grant permission to make the test set predictions public to facilitate replication/comparison by other researchers.
- [ ] We will release our system at: (specify URL)

**Comment by nschneid** [comment]
By submitting your system's test set predictions, you grant permission to the organizers to make those public. (If there are institutional barriers to this, please contact the organizers.)

**Comment by nschneid** [comment]
We will provide gold/mapped-from-gold-PTB universal POS tags at test time as well as training. #2

**Comment by nschneid** [comment]
According to @brendano, Yelp Academic Dataset may have changed, so we should specify the size to ensure everyone is using the same one.

> FYI the version i downloaded on 20150715 has
> 
> 474434 records total, within which there are
> 13490 businesses
> 330071 reviews
> 130873 users
> 
> you could just say "make sure the version of the data you use has 330071 reviews total" ...
> 
> (only the reviews have big blocks of complete sentences and narrative text.  the users and businesses both have name fields which conceivably might be of marginal use.)
